css/main.css

```diff
@@ -119,12 +119,173 @@ h1, h2 {
   margin-left: -15px !important;
 }
 
+.contact, .stats, .prospectus {
+  margin-top: 60px;
+}
+
+.info-input {
+  padding: 10px;
+  background-color: #FF6B98;
+}
+
+.inquiry-box {
+  border: 3px solid #FF6B98;
+}
+
+.form-control {
+  border-radius: 0;
+  border: none;
+}
+
+.inquiry-btn {
+  border-radius: 0;
+  background-color: #FF6B98;
+}
+
+.inquiry-btn:hover {
+  filter: brightness(1.2);
+  transition: .3s ease;
+}
+
+.overall-stats .heading {
+  font-size: 3.75rem;
+}
+
+.stat-number {
+  font-size: 4.3rem;
+  font-weight: bold;
+  line-height: 1;
+  color: #B6A1C4;
+}
+
+.stat-text {
+  font-size: 2rem;
+  line-height: 1;
+}
+
+.overall-stats img {
+  height: 100%;
+  width: 100%;
+  object-fit: cover;
+  box-shadow: -2.25rem 0rem 0 -20px #B6A1C4, 1rem -1rem #FF7BAC;
+}
+
+.demographics {
+  margin-top: 30px;
+  font-size: .9vw;
+}
+
+.demographics .heading {
+  font-size: 2.5rem;
+  color: #B6A1C4;
+  margin-bottom: 10px;
+}
+
+.demo-section {
+  background-color: #f3f3f3;
+  padding: 5%;
+  overflow: hidden; //because of floating stuff
+  height: 90%;
+  line-height: 1.75;
+}
+
+.stretch {
+  position:absolute;
+  top:0;
+  left:0;
+  right:0;
+  bottom:0;
+}
+
+.demo-name {
+  text-align: right;
+  padding-right: 5%;
+  float: left;
+  width: 85%;
+}
+
+.demo-stat {
+  text-align: left;
+  float: right;
+  width: 15%;
+}
+
+.edu {
+  line-height: 3.5;
+}
+
+.edu-dot {
+  width: 1.5vw;
+  height: 1.5vw;
+  border-radius: 50px;
+  display: inline-block;
+  vertical-align: -13%;
+  margin-right: 10px;
+}
+
+.edu .demo-stat {
+  color: grey;
+}
+
+td {
+  border-top: none !important;
+}
+
+th {
+  border-top: none !important;
+  padding-bottom: 5px !important;
+  border-bottom: solid 1px black;
+  background-color: white !important;
+}
+
+.add-ons td {
```

Comment:
If this CSS is different than the sleeper site css, can we move it to a separate file? that way the file is smaller and it's more manageable.

Reply:
yeah that makes sense, so for sponsor.html i would be importing both main.css and a separate sheet that has solely css for this page?